.gitignore

```diff
@@ -1,8 +1,2 @@
-pkg
-bin
-hound.sublime-workspace
-data
 .vagrant
-pub/index.html
-pub/excluded_files.html
-config.json
```

Comment:
I mention [workspace](https://golang.org/doc/code.html) casually, but I'm talking the "workspace" taken from the Go docs.